game.py

```diff
@@ -14,12 +17,33 @@
 
 
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
+
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        print("Wrong guesses so far:", wrong_guesses_list)
+        print_snowman_graphic(len(wrong_guesses_list))
```

Comment:
👍 Nice job using finding the length of `wrong_guesses_list` and passing that value to `print_snowman_graphic`. Doing so means you don't need to create an unnecessary variable like `wrong_guesses_count` that would need to be manually incremented (which would be a place that a bug could get introduced).